src/LocateDependencies/LocateDependenciesViaComposer.php

```diff
@@ -62,8 +63,6 @@ private function runInDirectory(callable $callable, string $directoryOfExecution
     {
         $originalDirectory = getcwd();
 
-        Assert::that($originalDirectory)->string();
```

Comment:
`getcwd` may return `false`, so assertion is needed

Reply:
Similar to above, now using `Safe\getcwd`